RELS-EXT_to_solr.xslt

```diff
@@ -38,13 +38,14 @@
     <xsl:template match="*[normalize-space(.)]" mode="rels_ext_element">
       <xsl:param name="prefix"/>
       <xsl:param name="suffix"/>
-
-      <xsl:call-template name="rels_ext_fields">
-        <xsl:with-param name="prefix" select="$prefix"/>
-        <xsl:with-param name="suffix" select="$suffix"/>
-        <xsl:with-param name="type">literal</xsl:with-param>
-        <xsl:with-param name="value" select="text()"/>
-      </xsl:call-template>
+      <xsl:if test="string($index_compound_sequence) = 'true' or (string($index_compound_sequence) = 'false' and not(self::islandora-rels-ext:* and starts-with(local-name(), 'isSequenceNumberOf')))">
```

Comment:
`islandora-rels-ext` namespace is undefined... needs a:
```xml
    xmlns:islandora-rels-ext="http://islandora.ca/ontology/relsext#"
```
somewhere above here in the XML tree.

Reply:
[Addressed](https://github.com/discoverygarden/islandora_transforms/pull/21/commits/a583830965da2cb38eaf9e4ed2570059e9599fd3#diff-558d2b79b209715d05ebfb7e32f5a47dR8).